game.py

```diff
@@ -20,8 +20,35 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
-
+    # Initialize the correct_letter_guess_statuses dictionary
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
+
+    # Loop until the player wins or loses
+    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES:
+        print("\nWord: " + generate_word_progress_string(snowman_word, correct_letter_guess_statuses))
+        print("Wrong guesses:", " ".join(sorted(wrong_guesses_list)))
+        # Get a letter from the user
+        user_input = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+        # Check if the user input is in the word
+        if user_input in correct_letter_guess_statuses:
```

Comment:
There is nothing inherently wrong with this particular conditional, but I do wonder what led you to check for the letter in `correct_letter_guess_statuses` rather than just the `snowman_word`. Technically they both have the same letters but if we follow the logic of the game, we'll want to check against the word itself rather than a dictionary that has been created by the word. This conditional closely mirrors a conditional in the function called on line 32 so comparing it to `snowman_word` directly avoids that duplication.